### PR TITLE
Add python API for im2sequence op

### LINKED ISSUE #7594: Add python API for block expand op
Add python API for block expand op

### PR DESCRIPTION
**Pull request by wanghaoshuang**
Add python API for im2sequence op
fix  #7594
Notice:  this pr depend on https://github.com/PaddlePaddle/Paddle/pull/4866

### CODE REVIEW

**Comment by JiayiFeng** [comment]
Do we have to use the name `block_expand`? `Block` is a core concept in the Fluid. This `op` seems to have nothing to do with it. So such a name is a little confusing.

**Comment by wanghaoshuang** [comment]
@JiayiFeng Thx.
We will discuss the naming of this operator. And it is being reviewed.
https://github.com/PaddlePaddle/Paddle/pull/4866#discussion_r161977405

File: python/paddle/v2/fluid/layers/nn.py [hunk 1]
```diff
@@ -2177,3 +2178,130 @@ def transpose(x, perm, name=None):
...
         attrs={'axis': perm})
     return out
+
+
+def im2sequence(input,
+                filter_size=1,
+                stride=1,
+                padding=0,
+                name=None,
+                layer_attr=None):
```
Comment (JiayiFeng) [inline python/paddle/v2/fluid/layers/nn.py]: `param_attr`?

Comment (JiayiFeng) [inline python/paddle/v2/fluid/layers/nn.py]: `param_attr`?

Comment (wanghaoshuang) [reply python/paddle/v2/fluid/layers/nn.py]: Thx. Removed  layer_attr / param_attr because this hasn't parameter.

**Pull request merged**